<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Git::Internal::SettingsPage</class>
 <widget class="QWidget" name="Git::Internal::SettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>403</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="configurationGroupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Configuration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="pathlabel">
        <property name="text">
         <string>Prepend to PATH:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="pathLineEdit"/>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QCheckBox" name="winHomeCheckBox">
        <property name="text">
         <string>Set &quot;HOME&quot; environment variable</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="noteLabel">
        <property name="text">
         <string>&lt;b&gt;Note:&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLabel" name="noteFieldlabel">
        <property name="text">
         <string>Git needs to find Perl in the environment.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Miscellaneous</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="pullRebaseCheckBox">
        <property name="text">
         <string>Pull with rebase</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>211</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="timeoutSpinBox">
        <property name="suffix">
         <string>s</string>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="logCountLabel">
        <property name="text">
         <string>Log count:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="timeoutLabel">
        <property name="text">
         <string>Timeout:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="logCountSpinBox">
        <property name="toolTip">
         <string>Note that huge amount of commits might take some time.</string>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gitkGroupBox">
     <property name="title">
      <string>Gitk</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="gitkOptionsLabel">
        <property name="text">
         <string>Arguments:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="gitkOptionsLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="repBrowserGroupBox">
     <property name="title">
      <string>Repository Browser</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="repBrowserCommandLabel">
        <property name="text">
         <string>Command:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Utils::PathChooser" name="repBrowserCommandPathChooser" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
   <slots>
    <signal>editingFinished()</signal>
    <signal>browsingFinished()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pathLineEdit</tabstop>
  <tabstop>winHomeCheckBox</tabstop>
  <tabstop>logCountSpinBox</tabstop>
  <tabstop>timeoutSpinBox</tabstop>
  <tabstop>pullRebaseCheckBox</tabstop>
  <tabstop>gitkOptionsLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
